arch/riscv/Kconfig

```diff
@@ -28,6 +28,16 @@ config RISCV_GP
 	  global pointer at program start or earlier than any instruction
 	  using GP relative addressing.
 
+config RISCV_CUSTOM_INTERRUPT_CONTROLLER
+	bool
+	help
+	  This option indicates that the ARM CPU is connected to a custom
```

Comment:
```suggestion
	  This option indicates that the CPU is connected to a custom
```

Not an ARM CPU ;)

Reply:
Thanks. I expect this commit to be removed soon after #63214 is merged. I added it as a quick fix to make my branch buildable since the solution in that PR is still being discussed.